api/src/main/java/org/motechproject/nms/api/web/KilkariController.java

```diff
@@ -67,12 +67,12 @@ public InboxResponse getInboxDetails(@RequestParam String callingNumber, @Reques
 
 
     private StringBuilder validateSaveInboxCallDetails(InboxCallDetailsRequest request) {
-        StringBuilder failureReasons = validate(request.getCallingNumber(), request.getOperator(),
-                request.getCircle(), request.getCallId());
+        StringBuilder failureReasons = validate(request.getCallingNumber(), request.getCallId(),
+                request.getOperator(), request.getCircle());
 
-        validateFieldNumeric(failureReasons, "callStartTime", request.getCallStartTime());
-        validateFieldNumeric(failureReasons, "callEndTime", request.getCallEndTime());
-        validateFieldNumeric(failureReasons, "callDurationInPulses", request.getCallDurationInPulses());
+        validateFieldPresent(failureReasons, "callStartTime", request.getCallStartTime());
```

Comment:
We expect time related fields to be in seconds since the epoch, should we validate that and then convert to DateTime in the api layer?